core/domain/opportunity_services.py

```diff
@@ -629,6 +629,28 @@ def get_exploration_opportunity_summaries_by_ids(
     return opportunities
 
 
+def get_exploration_opportunity_summary_by_id(
+    opportunity_id: str
+) -> Optional[opportunity_domain.ExplorationOpportunitySummary]:
+    """Returns an ExplorationOpportunitySummary object corresponding to the
+    opportunity id.
+
+    Args:
+        opportunity_id: str. An opportunity id.
+
+    Returns:
+        ExplorationOpportunitySummary|None. An ExplorationOpportunitySummary
+        domain object corresponding to the opportunity id if exists else None.
```

Comment:
Please see my previous [comment](https://github.com/oppia/oppia/pull/15917#discussion_r961967123) again. You're missing an "it" and a comma.

Reply:
Done